### PR TITLE
Adding semicolon separated list notes for multi settings

### PR DESCRIPTION
**Pull request by tobyash86**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -27,13 +27,13 @@ inputs:
     required: false
     default: 'builtin://Recommended .NET Core Rules'
   resource:
-    description: 'Solution-relative path to a project in a solution, a directory of files in a project, or a file.'
+    description: 'Solution-relative path to a project in a solution, a directory of files in a project, or a file. Use semicolon-separated list to specify many.'
```
Comment (annastuchlik) [inline action.yml:30]: Use a semicolon-separated list to specify multiple paths.

File: action.yml [hunk 2]
```diff
@@ -27,13 +27,13 @@ inputs:
...
     required: false
   include:
-    description: 'File system paths to files to include in analysis. Supports ANT-style wildcards. If not specified, all files are analyzed.'
+    description: 'File system paths to files to include in analysis. Supports ANT-style wildcards. If not specified, all files are analyzed. Use semicolon-separated list to specify many.'
```
Comment (annastuchlik) [inline action.yml:33]: Use a semicolon-separated list to specify multiple paths.

File: action.yml [hunk 3]
```diff
@@ -27,13 +27,13 @@ inputs:
...
     required: false
   exclude:
-    description: 'File system paths to files to exclude from analysis. Supports ANT-style wildcards.'
+    description: 'File system paths to files to exclude from analysis. Supports ANT-style wildcards. Use semicolon-separated list to specify many.'
```
Comment (annastuchlik) [inline action.yml:36]: Use a semicolon-separated list to specify multiple paths.

File: action.yml [hunk 4]
```diff
@@ -46,10 +46,10 @@ inputs:
     description: 'Publishes report to DTP.'
     required: false
   settings:
-    description: 'Path to a settings file.'
+    description: 'Path to a settings file. Use semicolon-separated list to specify many.'
```
Comment (annastuchlik) [inline action.yml:49]: Use a semicolon-separated list to specify multiple paths.

File: action.yml [hunk 5]
```diff
@@ -46,10 +46,10 @@ inputs:
...
     required: false
   property:
-    description: 'A single configuration setting in the "key=value" format.'
+    description: 'A single configuration setting in the "key=value" format. Use semicolon-separated list to specify many.'
```
Comment (annastuchlik) [inline action.yml:52]: Use a semicolon-separated list to specify multiple settings.

**Review by annastuchlik — CHANGES_REQUESTED** [review]

File: action.yml (hunk 1 above)
Comment (tobyash86) [reply action.yml]: Done.

**Comment by annastuchlik** [comment]
The changes look good. Thanks.

**Pull request merged**